src/admin-views/editor/column-body-price.php

```diff
@@ -13,7 +13,7 @@
 		echo $provider_obj->get_price_html( $ticket->ID );
 	} else {
 		if ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) {
-			esc_html_e( 'RSVP', 'event-tickets' );
+			echo tribe_get_rsvp_label_singular( basename( __FILE__ ) );
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 6f9fd370d012bc22b7124c85cdf81774</sub>